pkgs/development/tools/loganalyzer/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchurl
+, qtbase
+, qtsvg
+, wrapQtAppsHook
+}:
+
+let
+  pname = "loganalyzer";
+  appname = "LogAnalyzer";
+  version = "23.5.0";
+in
+stdenv.mkDerivation {
+  inherit pname appname version;
+
+  src = fetchurl {
+    url = "https://github.com/pbek/${pname}/releases/download/v${version}/source.tar.gz";
+    hash = "sha256-vmtUNU3w/iNoGCUyZgxup5kXqwb+o+0foQa/bd6IhAE=";
+  };
+
+  buildInputs = [
+    qtbase
+    qtsvg
+  ];
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+  ];
+
+  buildPhase = ''
+    qmake ${appname}.pro CONFIG+=release
```

Comment:
You can define `qmakeFlags` to avoid overwriting `buildPhase`.

Reply:
I'm trying now to use:

```
  qmakeFlags = [
    "${appname}.pro"
    "CONFIG+=release"
  ];
```

But I don't think `make` is executed:

```
nix-build -E '((import <nixpkgs> {}).libsForQt5.callPackage (import ./default.nix) { })'
this derivation will be built:
  /nix/store/ym2dy9hpxmsvvr5zh6swpsl1q8kysby8-loganalyzer-23.5.0.drv
building '/nix/store/ym2dy9hpxmsvvr5zh6swpsl1q8kysby8-loganalyzer-23.5.0.drv'...
qtPreHook
unpacking sources
unpacking source archive /nix/store/0ri04qr0v7bykbz6siylnlbp1512wa7i-source.tar.gz
source root is src
setting SOURCE_DATE_EPOCH to timestamp 1684414596 of file src/version.h
patching sources
configuring
no configure script, doing nothing
building
no Makefile or custom buildPhase, doing nothing
glibPreInstallPhase
glibPreInstallPhase
installing
install: creating directory '/nix/store/6rx5yh5gpfi50pnidivin1qa1nqi4xnj-loganalyzer-23.5.0'
install: creating directory '/nix/store/6rx5yh5gpfi50pnidivin1qa1nqi4xnj-loganalyzer-23.5.0/bin'
install: cannot stat 'LogAnalyzer': No such file or directory
error: builder for '/nix/store/ym2dy9hpxmsvvr5zh6swpsl1q8kysby8-loganalyzer-23.5.0.drv' failed with exit code 1;
       last 10 log lines:
       > configuring
       > no configure script, doing nothing
       > building
       > no Makefile or custom buildPhase, doing nothing
       > glibPreInstallPhase
       > glibPreInstallPhase
       > installing
       > install: creating directory '/nix/store/6rx5yh5gpfi50pnidivin1qa1nqi4xnj-loganalyzer-23.5.0'
       > install: creating directory '/nix/store/6rx5yh5gpfi50pnidivin1qa1nqi4xnj-loganalyzer-23.5.0/bin'
       > install: cannot stat 'LogAnalyzer': No such file or directory
       For full logs, run 'nix log /nix/store/ym2dy9hpxmsvvr5zh6swpsl1q8kysby8-loganalyzer-23.5.0.drv'.
make: *** [Makefile:2: build] Error 1
```

Reply:
Sorry, I missed the `make` command.
Then, could you please add "runHook" like installPhase?

Reply:
Thank you